### PR TITLE
nominate kerthcet as approver

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?
/kind documentation
/assign @ahg-g 

#### Which issue(s) this PR fixes

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -3,3 +3,9 @@
 approvers:
   - ahg-g
   - liurupeng
+  - kerthcet
+
+reviewers:
+  - ahg-g
+  - liurupeng
```
Comment (ahg-g) [inline OWNERS:10]: we may not be able to add Rupeng until he becomes a member

Comment (kerthcet) [reply OWNERS]: Remove from approvers as well, sorry for that Rupeng.

**Pull request merged**